README.md

```diff
@@ -1,30 +1,35 @@
+# Overview
 
-The other collection, ‘Mollie v2 API - OAuth/access_token authentication’, is based on access_token credentials and includes the Mollie Connect feature set to retrieve those. Thus, this collection is tailored for SaaS / marketplace setups and customers/merchants that want to operate on account-level (Onboarding API, Administration API, etc.)
+This repository contains a set of Postman v2.1 collections that you can use to try out the functionality of the Mollie API. You can import the JSON files into any local Postman instance. The following collections are available:
 
-## STRUCTURE
-The collection is structured like Mollie’s public API documentation. Every endpoint is represented in a call and every call contains one or more examples. The example set will be extended and aims at showcasing the variety of options the endpoint offers, rather than showing every possibility.
+* The **Mollie v2 API - API Key authentication** collection is based on API key authentication, and includes all endpoints that are available through that type of credentials. This collection is most applicable for a situation where there’s a one-to-one relation between a customer or merchant and Mollie, for example when one ecommerce platform instance connects to one Mollie account.
 
-## COLLECTION VARIABLES
-Most endpoint payloads contain variables instead of static values. All these variables are initiated at the Collection level and contain dummy data that can easily be adjusted. For the saved examples it was decided to display the actual values used.
+* The **Mollie v2 API - OAuth-access_token authentication** collection is based on `access_token` credentials, and includes the [Mollie Connect](https://docs.mollie.com/connect/overview) feature set to retrieve those access tokens. This collection is tailored for SaaS or marketplace setups and customers or merchants who want to operate on an account level, for example using Mollie's [Onboarding API](https://docs.mollie.com/reference/v2/onboarding-api/overview), Administration API, etc.
 
-## CREDENTIALS
-No credentials are present in the JSON files. Instead, everywhere credentials are required, variables are used instead. Postman supports authentication inheritance, so these variables are often called on higher-level objects (collection or folder). It is advised to create a Postman Environment and initiate all credentials in there. This prevents credentials from appearing in the collection itself. Besides, you can easily create multiple environments and use this to switch between Mollie accounts or App credentials.
+## Structure
+The structure of the collections follows the structure of Mollie’s public API documentation. A request is sent to each endpoint, and each request contains at least one example. The example set aims for showcasing the variety of options an endpoint offers, rather than showing every possibility.
```

Comment:
```suggestion
The structure of the collections follows the structure of Mollie’s public API documentation. A request is sent to each endpoint, and each request contains at least one example. The example set aims to showcase the variety of options an endpoint offers, rather than showing every possibility.
```